### ISSUE TITLE
Cannot reproduce the results in the paper

### ISSUE THREAD

**Issue by CurryYuan** [zyang-ur/SAT#1]
I run the training code with command provided in README, the results are lower than that in the paper.

| Accuracy  | SAT  | Reproduce |
| :-------: | :--: | :-------: |
|   Nr3d    | 49.2 |   46.0    |
| ScanRefer | 53.8 |   51.4    |

**Comment by jianghaojun** [comment]
Do you have any idea about where the gap comes from?

**Comment by clorislili** [comment]
Hi, is there any command for ScanRefer? CAN not find in README. Thanks!
